pxtblocks/fields/field_note.ts

```diff
@@ -114,6 +149,16 @@ namespace pxtblockly {
                 this.colour_ = pxtblockly.parseColour(params.editorColour);
                 this.colourBorder_ = goog.color.rgbArrayToHex(goog.color.darken(goog.color.hexToRgb(this.colour_), 0.2));
             }
+
+            if (params.minNote && params.maxNote) {
+                let minNote = parseInt(params.minNote);
```

Comment:
You already have defaults for min and max, why not allow an author to only set one.

Reply:
Agreed, it would be great if they were independent settings

Reply:
Good point, fixed.